packages/react/src/primitives/SearchField/SearchField.tsx

```diff
@@ -63,22 +63,24 @@ const SearchFieldPrimitive: Primitive<SearchFieldProps, 'input'> = (
     </FieldGroupIcon>
   ) : undefined;
 
+  const ClearButton = !isDisabled ? (
+    <FieldClearButton
+      ariaLabel={clearButtonLabel}
+      excludeFromTabOrder
+      isVisible={strHasLength(composedValue)}
```

Comment:
if it's more logical, I can rewrite using this isVisible prop
```suggestion
      isVisible={!isDisabled && strHasLength(composedValue)}
```

Reply:
I prefer the isVisible prop implementation, makes it easier to read for me. Want to hear what others think.